packages/x-charts-pro/src/ChartsToolbarPro/ChartsToolbarPro.tsx

```diff
@@ -26,22 +58,118 @@ export function ChartsToolbarPro(props: ChartsToolbarProps) {
     children.push(
       <Tooltip key="zoom-in" {...slotProps.baseTooltip} title={localeText.zoomIn}>
         <ChartsToolbarZoomInTrigger render={<ToolbarButton />}>
-          <ZoomInIcon {...slotProps.zoomInIcon} />
+          <ZoomInIcon fontSize="small" {...slotProps.zoomInIcon} />
         </ChartsToolbarZoomInTrigger>
       </Tooltip>,
     );
     children.push(
       <Tooltip key="zoom-out" {...slotProps.baseTooltip} title={localeText.zoomOut}>
         <ChartsToolbarZoomOutTrigger render={<ToolbarButton />}>
-          <ZoomOutIcon {...slotProps.zoomOutIcon} />
+          <ZoomOutIcon fontSize="small" {...slotProps.zoomOutIcon} />
         </ChartsToolbarZoomOutTrigger>
       </Tooltip>,
     );
   }
 
+  if (showExportMenu) {
+    const Tooltip = slots.baseTooltip;
+    const MenuList = slots.baseMenuList;
+    const MenuItem = slots.baseMenuItem;
+    const ExportIcon = slots.exportIcon;
+
+    const closeExportMenu = () => setExportMenuOpen(false);
+
+    const handleListKeyDown = (event: React.KeyboardEvent) => {
+      if (event.key === 'Tab') {
+        event.preventDefault();
+      }
+
+      if (isHideMenuKey(event.key)) {
+        closeExportMenu();
+      }
+    };
+
+    if (children.length > 0) {
+      children.push(<ChartsToolbarDivider key="divider" />);
+    }
+
+    children.push(
+      <React.Fragment key="export-menu">
+        <Tooltip title={localeText.toolbarExport}>
+          <ToolbarButton
+            ref={exportMenuTriggerRef}
+            id={exportMenuTriggerId}
+            aria-controls={exportMenuId}
+            aria-haspopup="true"
+            aria-expanded={exportMenuOpen ? 'true' : undefined}
+            onClick={() => setExportMenuOpen(!exportMenuOpen)}
+          >
+            <ExportIcon fontSize="small" />
+          </ToolbarButton>
+        </Tooltip>
+
+        <ChartsMenu
+          target={exportMenuTriggerRef.current}
+          open={exportMenuOpen}
+          onClose={closeExportMenu}
+          position="bottom-end"
+        >
```

Comment:
Not sure I like the visuals/idea of having the menu here 🤔

It feels a bit visually simpler, but it also hides the features away a bit, and adds to the amount of stuff we have to maintain.

I would expect at this stage to just have the buttons like you did in the "composition" example 🤔

Reply:
Given that up to 4 export formats are possible (print, PNG, JPEG, WebP), wouldn't that be too many buttons? 

Also, this follows the designs and the data grid. 

I don't think having buttons would work here, to be honest 🤔

Reply:
> Given that up to 4 export formats are possible (print, PNG, JPEG, WebP), wouldn't that be too many buttons?

It seems right now the menu only has "print & png", so I would expect us to only have two buttons, "print" and "image", with image being png by default?

Is this something the users can't update from a prop?

Reply:
Not a big issue though, just a proposition 😄